docs/api/misc_pretrained_weights.csv

```diff
@@ -1,2 +1,4 @@
 Weight,Channels,Source,Citation
 ResNet50_Weights.FMOW_RGB_GASSL, 3,`link <https://github.com/sustainlab-group/geography-aware-ssl>`__,`link <https://arxiv.org/abs/2011.09980>`__
+Swin_V2_B_Weights.NAIP_RGB_SATLAS,3,`link <https://github.com/allenai/satlas>`__,`link <https://arxiv.org/abs/2211.15660>`__
+Swin_V2_B_Weights.SENTINEL2_RGB_SATLAS,3,`link <https://github.com/allenai/satlas>`__,`link <https://arxiv.org/abs/2211.15660>`__
```

Comment:
If we know what satellite they were trained on, these should have their own categories/files. So Sen 2 should go under Sen 2, and we should have a new category/file for NAIP.